src/drivers/config.ts

```diff
@@ -9,15 +9,17 @@ This file is part of @p0security/cli
 You should have received a copy of the GNU General Public License along with @p0security/cli. If not, see <https://www.gnu.org/licenses/>.
 **/
 import { Config, RawOrgData } from "../types/org";
-import { P0_PATH } from "../util";
 import { bootstrapConfig } from "./env";
 import { bootstrapDoc } from "./firestore";
 import { print2 } from "./stdio";
 import { getDoc } from "firebase/firestore";
 import fs from "fs/promises";
+import os from "os";
 import path from "path";
+import process from "process";
 
-export const CONFIG_FILE_PATH = path.join(P0_PATH, "config.json");
+const getConfigFilePath = () =>
+  path.join(os.tmpdir(), "p0", `config.json-${process.ppid}`);
```

Comment:
is the parent id always going to be the terminal we have open, do we have to login again _every_ time we open a new terminal even if it's for the same environment?

Reply:
jumped to the code before reading the description and yes we'll have to reauth every time we open a new terminal 🤔 . Idk if that will be that bad tho.